src/services/WolfDen.Application/DTOs/Attendence/DailyStatusDTO.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace WolfDen.Application.Requests.DTOs.Attendence
+{
+    public class DailyStatusDTO
+    {
+        public DateTime ArrivalTime { get; set; }
+        public DateTime DepartureTime { get; set; }
+        public int InsideHours { get; set; }
+        public string Status { get; set; }

```

Comment:
Better approach  will be to pass the statusId and resolving the name in frontend